### PR TITLE
added a short lifecycle descripton

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

**Comment by aweiteka** [comment]
Has this been revised? I recall we discussed having a single directory for the spec and using tags to define released version and master branch would track current unreleased draft spec

**Comment by vpavlin** [comment]
I remember @goern did not agree with the master carrying the changes and I am not sure we came to any final conclusion.

**Comment by goern** [comment]
@vpavlin @aweiteka @tradej pls review

File: LIFECYCLE.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+All versions of the specification will be available using their corresponding URL.
+
+In addition to the human readable HTML document, a JSON formated machine readable version of the specification will be published at the same URL path as the HTML document. The document name will be schema.json and may reference other files using the JSON DRAFT4 references.
+
+The normative machine readable Nulecule Specification document will be published at http://www.projectatomic.io/nulecule/spec/<major.minor.patch>/schema.json 
+
+## States
+
+The Nulecule Specification will have a certain set of releases, we will use semantic versioning to identify the releases. 
+Prior each release there will be a draft version of the release. This will be use to work/collaborate on the spec itself.
```
Comment (vpavlin) [inline LIFECYCLE.md]: will be used

Comment (goern) [inline LIFECYCLE.md]: fixed.

**Comment by aweiteka** [comment]
I like it.

**Pull request merged**